deployment/overlays/balloons/kustomization.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: kube-system
+
+images:
+- name: '*'
+  newName: nri-resmgr-balloons
+  newTag: devel
+
+bases:
+  - ../../base/crds
+  - ../../base/nri-resmgr
+
+resources:
+  - sample-configmap.yaml
```

let's drop bases and merge its content under the `resources`, because bases is deprecated. Ref: https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/bases/
```suggestion
resources:
  - ../../base/crds
  - ../../base/nri-resmgr
  - sample-configmap.yaml
```

---

Ach, good point, thanks, fixed